backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -154,54 +154,66 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
           // Try to make a Path of the callRootPath from the detritus
           val next = lookupSourceCallRootPath(jobDetritus) match {
             case Success(sourceCallRootPath) =>
-
-              // process simpletons and detritus to get updated paths and corresponding IoCommands
-              val processed = for {
-                (destinationCallOutputs, simpletonIoCommands) <- processSimpletons(simpletons, sourceCallRootPath)
-                (destinationDetritus, detritusIoCommands) <- processDetritus(jobDetritus)
-              } yield (destinationCallOutputs, destinationDetritus, simpletonIoCommands ++ detritusIoCommands)
-
-              processed match {
-                case Success((destinationCallOutputs, destinationDetritus, detritusAndOutputsIoCommands)) =>
-                  duplicate(ioCommandsToCopyPairs(detritusAndOutputsIoCommands)) match {
-                    // Use the duplicate override if exists
-                    case Some(Success(_)) => succeedAndStop(returnCode, destinationCallOutputs, destinationDetritus)
-                    case Some(Failure(failure)) =>
-                      // Something went wrong in the custom duplication code. We consider this loggable because it's most likely a user-permission error:
-                      failAndStop(CopyAttemptError(failure))
-                    // Otherwise send the first round of IoCommands (file outputs and detritus) if any
-                    case None if detritusAndOutputsIoCommands.nonEmpty =>
-                      detritusAndOutputsIoCommands foreach sendIoCommand
-
-                      // Add potential additional commands to the list
-                      val additionalCommandsTry =
-                        additionalIoCommands(
-                          sourceCallRootPath = sourceCallRootPath,
-                          originalSimpletons = simpletons,
-                          newOutputs = destinationCallOutputs,
-                          originalDetritus = jobDetritus,
-                          newDetritus = destinationDetritus,
-                        )
-                      additionalCommandsTry match {
-                        case Success(additionalCommands) =>
-                          val allCommands = List(detritusAndOutputsIoCommands) ++ additionalCommands
-                          goto(WaitingForIoResponses) using
-                            Option(StandardCacheHitCopyingActorData(
-                              commandsToWaitFor = allCommands,
-                              newJobOutputs = destinationCallOutputs,
+              log.warning("willy, about to process locations")
+              val processedLocation = for {
+                (ignoreThis, simpletonLocationCheckCommands) <- processSimpletonLocations(simpletons, sourceCallRootPath)
```

Comment:
an underscore in this context is Scala for "ignore this" 🙂 
```suggestion
                (_, simpletonLocationCheckCommands) <- processSimpletonLocations(simpletons, sourceCallRootPath)
```